lib/routes/devices-and-sessions.js

```diff
@@ -606,19 +606,28 @@ module.exports = (log, db, config, customs, push, pushbox, devices) => {
       handler: async function (request) {
         log.begin('Account.deviceDestroy', request)
 
-        const sessionToken = request.auth.credentials
-        const uid = sessionToken.uid
+        const credentials = request.auth.credentials
+        const uid = credentials.uid
         const id = request.payload.id
         let devices
 
-        // We want to inculde the disconnected device in the list
+        // We want to include the disconnected device in the list
         // of devices to notify, so list them before disconnecting.
         return request.app.devices
           .then(res => {
             devices = res
             return db.deleteDevice(uid, id)
           })
-          .then(res => {
+          .then(() => {
+            const deviceToDelete = devices.find(d => d.id === id)
```

Comment:
This'll work, but the db-server PR deliberately returns the `refreshTokenId` of the device it deleted:

  https://github.com/mozilla/fxa-auth-db-mysql/pull/483/files#diff-e353759afb5be2d3d029466ce036d212R109

So you shouldn't have to do a separate `.find()` here.  Is the code in `./lib/db.js` somehow swallowing that return value before it gets to your code here?

Reply:
> somehow swallowing that return value

That is the case, I shall file a follow up issue for this

Reply:
Filed https://github.com/mozilla/fxa-auth-server/issues/2978